### ISSUE TITLE
Use of uninitialized variable

### ISSUE THREAD

**Issue by ortem** [intellij-rust/intellij-rust#3162]
Add [E0381](https://doc.rust-lang.org/error-index.html#E0381) diagnostic to borrow checker (looks like this is about handling `MoveKind.Declared` case inside `RsBorrowCheckerInspection`)
```rust
fn main() {
    let x: i32;
    let y = x; // error, use of possibly uninitialized variable
}
```

**Comment by gagan0723** [comment]
Hello @ortem, the plugin still does not provide any errors in case of uninitialized variables.

## Environment

* **Intellij-Rust plugin version: v0.2.94.2119-183(latest)**
* **Rust toolchain version: rustc 1.33.0 (2aa4c46cf 2019-02-28) (stable)**
* **IDE name and version: CLion 2018.3.4**
* **Operating system: macOS Mojave Version 10.14.4**

## Problem description
Plugin does not show any errors when uninitialized variable is used.
